locksmith/src/operations/userOperations.ts

```diff
@@ -1,5 +1,7 @@
 import { UserCreationInput } from '../types' // eslint-disable-line no-unused-vars
-import * as Normalizer from '../utils/normalizer' // eslint-disable-line no-unused-vars
+import * as Normalizer from '../utils/normalizer'
+// eslint-disable-line no-unused-vars
```

Comment:
is that eslint being sily?

Reply:
yeah, I am not sure what that is about. I'll see if I can save it without it being re-adjusted

Reply:
Is this just barely over the line-length limit? That's annoying. You can use disable-next-line (or something like that) if it persists.